[2022-09-05 11:05:47,146][__main__][INFO] - device: cuda:0
[2022-09-05 11:05:47,146][deepke.relation_extraction.standard.tools.preprocess][INFO] - ===== start preprocess data =====
[2022-09-05 11:05:47,147][deepke.relation_extraction.standard.tools.preprocess][INFO] - load raw files...
[2022-09-05 11:05:47,147][utils.ioUtils][INFO] - load csv from /home/zzx/deepke/DeepKE/example/re/standard/data/origin/train.csv
Traceback (most recent call last):
  File "run.py", line 51, in main
    preprocess(cfg)
  File "/home/zzx/deepke/DeepKE/src/deepke/relation_extraction/standard/tools/preprocess.py", line 189, in preprocess
    train_data = load_csv(train_fp)
  File "/home/zzx/deepke/DeepKE/src/deepke/attribution_extraction/standard/utils/ioUtils.py", line 70, in load_csv
    with open(fp, encoding='utf-8') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/zzx/deepke/DeepKE/example/re/standard/data/origin/train.csv'
Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.